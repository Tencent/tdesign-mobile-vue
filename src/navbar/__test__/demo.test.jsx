/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import { mount } from '@vue/test-utils';
import baseVue from '@/navbar/demos/base.vue';
import customColorVue from '@/navbar/demos/custom-color.vue';
import imgVue from '@/navbar/demos/img.vue';
import leftTitleVue from '@/navbar/demos/left-title.vue';
import mobileVue from '@/navbar/demos/mobile.vue';
import searchVue from '@/navbar/demos/search.vue';
import sizeVue from '@/navbar/demos/size.vue';

const mapper = {
  baseVue,
  customColorVue,
  imgVue,
  leftTitleVue,
  mobileVue,
  searchVue,
  sizeVue,
};

describe('Navbar', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Navbar ${demoName} demo works fine`, () => {
      const wrapper = mount(mapper[demoName]);
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
