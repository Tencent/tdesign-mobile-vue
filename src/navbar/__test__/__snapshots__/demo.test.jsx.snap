// Vitest Snapshot v1

exports[`Navbar > Navbar baseVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-navbar t-navbar--visible-animation"
    data-v-7388e034=""
    style="position: relative;"
  >
    <!---->
    <div
      class="t-navbar__content"
    >
      <div
        class="t-navbar__left"
      >
        <svg
          class="t-icon t-icon-chevron-left t-navbar__left-arrow"
          fill="none"
          height="1em"
          viewBox="0 0 24 24"
          width="1em"
        >
          <path
            d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
            fill="currentColor"
          />
        </svg>
        <!---->
        <!---->
      </div>
      <div
        class="t-navbar__center"
      >
        <span
          class="t-navbar__center-title"
        >
          标题文字
        </span>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="t-navbar t-navbar--visible-animation"
    data-v-7388e034=""
    style="position: relative;"
  >
    <!---->
    <div
      class="t-navbar__content"
    >
      <div
        class="t-navbar__left"
      >
        <svg
          class="t-icon t-icon-chevron-left t-navbar__left-arrow"
          fill="none"
          height="1em"
          viewBox="0 0 24 24"
          width="1em"
        >
          <path
            d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
            fill="currentColor"
          />
        </svg>
        
        <svg
          class="t-icon t-icon-close"
          data-v-7388e034=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-close"
          />
        </svg>
        
        <!---->
      </div>
      <div
        class="t-navbar__center"
      >
        <span
          class="t-navbar__center-title"
        >
          标题文字
        </span>
      </div>
      <div
        class="t-navbar__right"
      >
        
        <svg
          class="t-icon t-icon-ellipsis"
          data-v-7388e034=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-ellipsis"
          />
        </svg>
        
      </div>
    </div>
  </div>
  <div
    class="t-navbar t-navbar--visible-animation"
    data-v-7388e034=""
    style="position: relative;"
  >
    <!---->
    <div
      class="t-navbar__content"
    >
      <div
        class="t-navbar__left"
      >
        <!---->
        <!---->
        <div
          class="t-navbar__capsule"
        >
          
          <div
            class="custom-capsule"
            data-v-7388e034=""
          >
            <svg
              class="t-icon t-icon-chevron-left custom-capsule__icon"
              data-v-7388e034=""
              style="font-size: 20px;"
            >
              <use
                href="#t-icon-chevron-left"
              />
            </svg>
            <div
              class="t-divider t-divider--vertical t-divider--center"
              data-v-7388e034=""
              role="separator"
            >
              <div
                class="t-divider__content"
              >
                <!---->
              </div>
            </div>
            <svg
              class="t-icon t-icon-home custom-capsule__icon"
              data-v-7388e034=""
              style="font-size: 20px;"
            >
              <use
                href="#t-icon-home"
              />
            </svg>
          </div>
          
        </div>
      </div>
      <div
        class="t-navbar__center"
      >
        <span
          class="t-navbar__center-title"
        >
          标题文字超...
        </span>
      </div>
      <div
        class="t-navbar__right"
      >
        
        <svg
          class="t-icon t-icon-ellipsis"
          data-v-7388e034=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-ellipsis"
          />
        </svg>
        
      </div>
    </div>
  </div>
  
</div>
`;

exports[`Navbar > Navbar customColorVue demo works fine 1`] = `
<div
  class="t-navbar t-navbar--visible-animation custom-navbar"
  data-v-b4f331c8=""
  style="position: relative;"
>
  <!---->
  <div
    class="t-navbar__content"
  >
    <div
      class="t-navbar__left"
    >
      <svg
        class="t-icon t-icon-chevron-left t-navbar__left-arrow"
        fill="none"
        height="1em"
        viewBox="0 0 24 24"
        width="1em"
      >
        <path
          d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
          fill="currentColor"
        />
      </svg>
      <!---->
      <!---->
    </div>
    <div
      class="t-navbar__center"
    >
      <span
        class="t-navbar__center-title"
      >
        标题文字
      </span>
    </div>
    <div
      class="t-navbar__right"
    >
      
      <svg
        class="t-icon t-icon-ellipsis"
        data-v-b4f331c8=""
        style="font-size: 24px;"
      >
        <use
          href="#t-icon-ellipsis"
        />
      </svg>
      
    </div>
  </div>
</div>
`;

exports[`Navbar > Navbar imgVue demo works fine 1`] = `
<div
  class="t-navbar t-navbar--visible-animation"
  data-v-0848b63e=""
  style="position: relative;"
>
  <!---->
  <div
    class="t-navbar__content"
  >
    <div
      class="t-navbar__left"
    >
      <!---->
      
      <div
        class="t-image t-image--square logo"
        data-v-0848b63e=""
      >
        <div
          class="t-image__mask"
        >
          <div
            class="t-loading"
            style="font-size: 20px;"
          >
            <div
              class="t-loading__dots"
              style="animation-duration: 800ms; width: 20px; height: 20px;"
            >
              
              <div
                class="t-loading__dot"
                style="animation-duration: 0.8s; animation-delay: 0s;"
              />
              <div
                class="t-loading__dot"
                style="animation-duration: 0.8s; animation-delay: 0.26666666666666666s;"
              />
              <div
                class="t-loading__dot"
                style="animation-duration: 0.8s; animation-delay: 0.5333333333333333s;"
              />
              
            </div>
            <!--v-if-->
            <!---->
          </div>
        </div>
        <picture>
          <!--v-if-->
          <img
            alt=""
            class="t-image__img"
            src="https://tdesign.gtimg.com/mobile/demos/logo1.png"
            style="object-fit: fill; object-position: center;"
          />
        </picture>
      </div>
      
      <!---->
    </div>
    <div
      class="t-navbar__center"
    >
      <!---->
    </div>
    <div
      class="t-navbar__right"
    >
      
      <svg
        class="t-icon t-icon-home"
        data-v-0848b63e=""
        style="font-size: 24px;"
      >
        <use
          href="#t-icon-home"
        />
      </svg>
      <svg
        class="t-icon t-icon-ellipsis"
        data-v-0848b63e=""
        style="font-size: 24px;"
      >
        <use
          href="#t-icon-ellipsis"
        />
      </svg>
      
    </div>
  </div>
</div>
`;

exports[`Navbar > Navbar leftTitleVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-navbar t-navbar--visible-animation"
    data-v-63d9f832=""
    style="position: relative;"
  >
    <!---->
    <div
      class="t-navbar__content"
    >
      <div
        class="t-navbar__left"
      >
        <svg
          class="t-icon t-icon-chevron-left t-navbar__left-arrow"
          fill="none"
          height="1em"
          viewBox="0 0 24 24"
          width="1em"
        >
          <path
            d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
            fill="currentColor"
          />
        </svg>
        <!---->
        <!---->
      </div>
      <div
        class="t-navbar__center"
      >
        <span
          class="t-navbar__center-title"
        >
          标题居中
        </span>
      </div>
      <div
        class="t-navbar__right"
      >
        
        <svg
          class="t-icon t-icon-ellipsis"
          data-v-63d9f832=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-ellipsis"
          />
        </svg>
        
      </div>
    </div>
  </div>
  <div
    class="t-navbar t-navbar--visible-animation"
    data-v-63d9f832=""
    style="position: relative;"
  >
    <!---->
    <div
      class="t-navbar__content"
    >
      <div
        class="t-navbar__left"
      >
        <svg
          class="t-icon t-icon-chevron-left t-navbar__left-arrow"
          fill="none"
          height="1em"
          viewBox="0 0 24 24"
          width="1em"
        >
          <path
            d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
            fill="currentColor"
          />
        </svg>
        
        <span
          class="custom-title"
          data-v-63d9f832=""
        >
          标题左对齐
        </span>
        
        <!---->
      </div>
      <div
        class="t-navbar__center"
      >
        <!---->
      </div>
      <div
        class="t-navbar__right"
      >
        
        <svg
          class="t-icon t-icon-home"
          data-v-63d9f832=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-home"
          />
        </svg>
        <svg
          class="t-icon t-icon-ellipsis"
          data-v-63d9f832=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-ellipsis"
          />
        </svg>
        
      </div>
    </div>
  </div>
  
</div>
`;

exports[`Navbar > Navbar mobileVue demo works fine 1`] = `
<div
  class="tdesign-mobile-demo"
  data-v-16defdf7=""
>
  <h1
    class="title"
    data-v-16defdf7=""
  >
    NavBar 导航栏
  </h1>
  <p
    class="summary"
    data-v-16defdf7=""
  >
    用于不同页面之间切换或者跳转，位于内容区的上方，系统状态栏的下方。
  </p>
  <div
    class="tdesign-mobile-demo-block"
    data-v-16defdf7=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        01 组件类型
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        基础导航栏
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-7388e034=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <svg
              class="t-icon t-icon-chevron-left t-navbar__left-arrow"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
            >
              <path
                d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
                fill="currentColor"
              />
            </svg>
            <!---->
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <span
              class="t-navbar__center-title"
            >
              标题文字
            </span>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-7388e034=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <svg
              class="t-icon t-icon-chevron-left t-navbar__left-arrow"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
            >
              <path
                d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
                fill="currentColor"
              />
            </svg>
            
            <svg
              class="t-icon t-icon-close"
              data-v-7388e034=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-close"
              />
            </svg>
            
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <span
              class="t-navbar__center-title"
            >
              标题文字
            </span>
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-7388e034=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-7388e034=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <!---->
            <!---->
            <div
              class="t-navbar__capsule"
            >
              
              <div
                class="custom-capsule"
                data-v-7388e034=""
              >
                <svg
                  class="t-icon t-icon-chevron-left custom-capsule__icon"
                  data-v-7388e034=""
                  style="font-size: 20px;"
                >
                  <use
                    href="#t-icon-chevron-left"
                  />
                </svg>
                <div
                  class="t-divider t-divider--vertical t-divider--center"
                  data-v-7388e034=""
                  role="separator"
                >
                  <div
                    class="t-divider__content"
                  >
                    <!---->
                  </div>
                </div>
                <svg
                  class="t-icon t-icon-home custom-capsule__icon"
                  data-v-7388e034=""
                  style="font-size: 20px;"
                >
                  <use
                    href="#t-icon-home"
                  />
                </svg>
              </div>
              
            </div>
          </div>
          <div
            class="t-navbar__center"
          >
            <span
              class="t-navbar__center-title"
            >
              标题文字超...
            </span>
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-7388e034=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-16defdf7=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        带搜索导航栏
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-16defdf7=""
        data-v-6af0dffd=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <!---->
            
            <div
              class="t-search"
              data-v-6af0dffd=""
            >
              <div
                class="t-search__input-box t-search__input-box--round"
              >
                <svg
                  class="t-icon t-icon-search"
                  fill="none"
                  height="1em"
                  style="font-size: 24px;"
                  viewBox="0 0 24 24"
                  width="1em"
                >
                  <path
                    d="M15.1 5.9a6.5 6.5 0 10-9.2 9.2 6.5 6.5 0 009.2-9.2zM4.49 4.5a8.5 8.5 0 0112.69 11.27l5.34 5.35-1.41 1.41-5.35-5.34A8.5 8.5 0 014.5 4.49z"
                    fill="currentColor"
                  />
                </svg>
                <input
                  class="t-input__keyword"
                  placeholder="搜索预设文案"
                  type="search"
                />
                <!---->
              </div>
              <!---->
            </div>
            
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <!---->
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-home"
              data-v-6af0dffd=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-home"
              />
            </svg>
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-6af0dffd=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-16defdf7=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        带图片导航栏
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-0848b63e=""
        data-v-16defdf7=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <!---->
            
            <div
              class="t-image t-image--square logo"
              data-v-0848b63e=""
            >
              <div
                class="t-image__mask"
              >
                <div
                  class="t-loading"
                  style="font-size: 20px;"
                >
                  <div
                    class="t-loading__dots"
                    style="animation-duration: 800ms; width: 20px; height: 20px;"
                  >
                    
                    <div
                      class="t-loading__dot"
                      style="animation-duration: 0.8s; animation-delay: 0s;"
                    />
                    <div
                      class="t-loading__dot"
                      style="animation-duration: 0.8s; animation-delay: 0.26666666666666666s;"
                    />
                    <div
                      class="t-loading__dot"
                      style="animation-duration: 0.8s; animation-delay: 0.5333333333333333s;"
                    />
                    
                  </div>
                  <!--v-if-->
                  <!---->
                </div>
              </div>
              <picture>
                <!--v-if-->
                <img
                  alt=""
                  class="t-image__img"
                  src="https://tdesign.gtimg.com/mobile/demos/logo1.png"
                  style="object-fit: fill; object-position: center;"
                />
              </picture>
            </div>
            
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <!---->
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-home"
              data-v-0848b63e=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-home"
              />
            </svg>
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-0848b63e=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block"
    data-v-16defdf7=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        02 组件样式
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        标题对齐
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-63d9f832=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <svg
              class="t-icon t-icon-chevron-left t-navbar__left-arrow"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
            >
              <path
                d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
                fill="currentColor"
              />
            </svg>
            <!---->
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <span
              class="t-navbar__center-title"
            >
              标题居中
            </span>
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-63d9f832=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-63d9f832=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <svg
              class="t-icon t-icon-chevron-left t-navbar__left-arrow"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
            >
              <path
                d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
                fill="currentColor"
              />
            </svg>
            
            <span
              class="custom-title"
              data-v-63d9f832=""
            >
              标题左对齐
            </span>
            
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <!---->
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-home"
              data-v-63d9f832=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-home"
              />
            </svg>
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-63d9f832=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-16defdf7=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        标题尺寸
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="t-navbar t-navbar--visible-animation"
        data-v-5f535116=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <svg
              class="t-icon t-icon-chevron-left t-navbar__left-arrow"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
            >
              <path
                d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
                fill="currentColor"
              />
            </svg>
            <!---->
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <span
              class="t-navbar__center-title"
            >
              标题文字
            </span>
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-5f535116=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      <div
        class="t-navbar t-navbar--visible-animation custom-navbar"
        data-v-5f535116=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <svg
              class="t-icon t-icon-chevron-left t-navbar__left-arrow"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
            >
              <path
                d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
                fill="currentColor"
              />
            </svg>
            <!---->
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <!---->
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-home"
              data-v-5f535116=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-home"
              />
            </svg>
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-5f535116=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      <div
        class="header-title"
        data-v-5f535116=""
      >
        大尺寸标题
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-16defdf7=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        自定义颜色
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      <div
        class="t-navbar t-navbar--visible-animation custom-navbar"
        data-v-16defdf7=""
        data-v-b4f331c8=""
        style="position: relative;"
      >
        <!---->
        <div
          class="t-navbar__content"
        >
          <div
            class="t-navbar__left"
          >
            <svg
              class="t-icon t-icon-chevron-left t-navbar__left-arrow"
              fill="none"
              height="1em"
              viewBox="0 0 24 24"
              width="1em"
            >
              <path
                d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
                fill="currentColor"
              />
            </svg>
            <!---->
            <!---->
          </div>
          <div
            class="t-navbar__center"
          >
            <span
              class="t-navbar__center-title"
            >
              标题文字
            </span>
          </div>
          <div
            class="t-navbar__right"
          >
            
            <svg
              class="t-icon t-icon-ellipsis"
              data-v-b4f331c8=""
              style="font-size: 24px;"
            >
              <use
                href="#t-icon-ellipsis"
              />
            </svg>
            
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
`;

exports[`Navbar > Navbar searchVue demo works fine 1`] = `
<div
  class="t-navbar t-navbar--visible-animation"
  data-v-6af0dffd=""
  style="position: relative;"
>
  <!---->
  <div
    class="t-navbar__content"
  >
    <div
      class="t-navbar__left"
    >
      <!---->
      
      <div
        class="t-search"
        data-v-6af0dffd=""
      >
        <div
          class="t-search__input-box t-search__input-box--round"
        >
          <svg
            class="t-icon t-icon-search"
            fill="none"
            height="1em"
            style="font-size: 24px;"
            viewBox="0 0 24 24"
            width="1em"
          >
            <path
              d="M15.1 5.9a6.5 6.5 0 10-9.2 9.2 6.5 6.5 0 009.2-9.2zM4.49 4.5a8.5 8.5 0 0112.69 11.27l5.34 5.35-1.41 1.41-5.35-5.34A8.5 8.5 0 014.5 4.49z"
              fill="currentColor"
            />
          </svg>
          <input
            class="t-input__keyword"
            placeholder="搜索预设文案"
            type="search"
          />
          <!---->
        </div>
        <!---->
      </div>
      
      <!---->
    </div>
    <div
      class="t-navbar__center"
    >
      <!---->
    </div>
    <div
      class="t-navbar__right"
    >
      
      <svg
        class="t-icon t-icon-home"
        data-v-6af0dffd=""
        style="font-size: 24px;"
      >
        <use
          href="#t-icon-home"
        />
      </svg>
      <svg
        class="t-icon t-icon-ellipsis"
        data-v-6af0dffd=""
        style="font-size: 24px;"
      >
        <use
          href="#t-icon-ellipsis"
        />
      </svg>
      
    </div>
  </div>
</div>
`;

exports[`Navbar > Navbar sizeVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-navbar t-navbar--visible-animation"
    data-v-5f535116=""
    style="position: relative;"
  >
    <!---->
    <div
      class="t-navbar__content"
    >
      <div
        class="t-navbar__left"
      >
        <svg
          class="t-icon t-icon-chevron-left t-navbar__left-arrow"
          fill="none"
          height="1em"
          viewBox="0 0 24 24"
          width="1em"
        >
          <path
            d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
            fill="currentColor"
          />
        </svg>
        <!---->
        <!---->
      </div>
      <div
        class="t-navbar__center"
      >
        <span
          class="t-navbar__center-title"
        >
          标题文字
        </span>
      </div>
      <div
        class="t-navbar__right"
      >
        
        <svg
          class="t-icon t-icon-ellipsis"
          data-v-5f535116=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-ellipsis"
          />
        </svg>
        
      </div>
    </div>
  </div>
  <div
    class="t-navbar t-navbar--visible-animation custom-navbar"
    data-v-5f535116=""
    style="position: relative;"
  >
    <!---->
    <div
      class="t-navbar__content"
    >
      <div
        class="t-navbar__left"
      >
        <svg
          class="t-icon t-icon-chevron-left t-navbar__left-arrow"
          fill="none"
          height="1em"
          viewBox="0 0 24 24"
          width="1em"
        >
          <path
            d="M15.91 17.5l-5.5-5.5 5.5-5.5-1.41-1.41L7.59 12l6.91 6.91 1.41-1.41z"
            fill="currentColor"
          />
        </svg>
        <!---->
        <!---->
      </div>
      <div
        class="t-navbar__center"
      >
        <!---->
      </div>
      <div
        class="t-navbar__right"
      >
        
        <svg
          class="t-icon t-icon-home"
          data-v-5f535116=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-home"
          />
        </svg>
        <svg
          class="t-icon t-icon-ellipsis"
          data-v-5f535116=""
          style="font-size: 24px;"
        >
          <use
            href="#t-icon-ellipsis"
          />
        </svg>
        
      </div>
    </div>
  </div>
  <div
    class="header-title"
    data-v-5f535116=""
  >
    大尺寸标题
  </div>
  
</div>
`;
